kernels/cpu/src/actions/CPUEvaluateTopologyReactions.cpp

```diff
@@ -205,6 +215,7 @@ CPUEvaluateTopologyReactions::topology_reaction_events CPUEvaluateTopologyReacti
                 std::size_t reaction_idx = 0;
                 for (const auto &reaction : topology_types.structuralReactionsOf(top->type())) {
                     TREvent event{};
+                    event.reactionId = reaction.id();
                     event.rate = top->rates().at(reaction_idx);
                     event.cumulativeRate = event.rate + current_cumulative_rate;
                     current_cumulative_rate = event.cumulativeRate;
```

Comment:
naming conventions are a bit off in this function ;)

Reply:
consider `currentCumulativeRate` instead of `current_cumulative_rate`, and `TopologyReactionEvents` instead of `topology_reaction_events`